src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -624,7 +624,7 @@ private void updateUndrawnTiles(final List<Tile> undrawnTiles, final int preDraw
         if (tile.isDirty()) {
           undrawnTiles.add(tile);
         } else if (forceInMemory) {
-          images.add(tile.getRawImage());
+          images.add(tile);
```

Comment:
Not sure exactly what this is doing but caching the tile instead of raw image might not be good.